src/applications/ask-va/constants.js

```diff
@@ -4,6 +4,7 @@ export const envUrl = environment.API_URL;
 
 // Used to test against dev
 // export const envUrl = 'https://dev-api.va.gov';
+//
```

Comment:
Can this be cleaned up before merging?

Reply:
We keep this commented out since we use it often for testing api calls that can only be made on dev, it will definitely get removed as we get all our api calls in order.